shared/modules/servicer_module.go

```diff
@@ -0,0 +1,16 @@
+package modules
+
+//go:generate mockgen -destination=./mocks/servicer_module_mock.go github.com/pokt-network/pocket/shared/modules ServicerModule
+
+import (
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+)
+
+const (
+	ServicerModuleName = "servicer"
+)
+
+type ServicerModule interface {
+	Module
```

Comment:
Should this be a submodule under `utility` or is module the correct classification?

Reply:
Great point. Not 100% clear yet. Currently the servicer is a submodule of `utility` and this is not something introduced by this PR. So I would say this decision is out of scope for this PR.